core/java/iesi-core/src/main/java/io/metadew/iesi/script/execution/instruction/data/text/TextSubstring.java

```diff
@@ -73,6 +88,18 @@ private void verifyArguments(String text, int start) {
         }
     }
 
+    private int countNumberOfLines(String str, int index, int numberOfArguments){
+        if (numberOfArguments==2){
+            return str.substring(index).length() - str.substring(index).replace("\n","").length();
+        }
+        return str.substring(0,index).length() - str.substring(0,index).replace("\n","").length();
+    }
+
+    private int countNumberOfLinesInside(String str, int start, int end){
+        return str.substring(start,end).length() - str.substring(start,end).replace("\n","").length();
+    }
+
+
```

Comment:
formatting of code. In IntelliJ you can auto format the code. Normally SonarLint should also complain about this formatting. Is the plugin installed at your side?